README.md

```diff
@@ -1,24 +1,203 @@
-# README
+<!-- TABLE OF CONTENTS -->
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# 📗 Table of Contents
 
-Things you may want to cover:
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
```

Comment:
- [ ] Hi @Kelvin-Ben, I couldn't see your app live link. Have you tried to follow the [tutorials ](https://www.youtube.com/watch?v=8ofyx5slcug)@marco Lizza(Student Success) sent to us in slack? This may be helpful to you. We really need to make our app accessible all over the world :+1:

Reply:
@Deepakdanger, Thank you so much, this review really helped me. I am so greatful for you details and suggestions. I have implemented every change that you requested